sktime/forecasting/compose/_pipeline.py

```diff
@@ -122,6 +121,9 @@ def fit(self, y, X=None, fh=None):
         return self
 
     def _predict(self, fh=None, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+        # import here to avoid circular dependency
+        from sktime.utils._testing.estimator_checks import _has_tag
```

Comment:
you mean the function `_has_tag`?

Reply:
Yes!

Reply:
ok now I have another issue with `VALID_ESTIMATOR_TAGS`. I think the general issue is the `_conf.py` file because it contains all those imports but also the configurations. So I cant import the configurations without having a circular import. Can I put  `VALID_ESTIMATOR_TAGS` and `NON_STATE_CHANGING_METHODS` to another file?

Reply:
or I create a separate function somewhere to avoid calling `_has_tag()` in `_pipeline.py`?

Reply:
Please see my comments below - started a new thread :D